docs/victoriametrics/README.md

```diff
@@ -1850,6 +1850,8 @@ since it uses lower amounts of RAM, CPU and network bandwidth than Prometheus.
 If you use identically configured [vmagent](https://docs.victoriametrics.com/vmagent/) instances for collecting the same data
 and sending it to VictoriaMetrics, then do not forget enabling [deduplication](#deduplication) at VictoriaMetrics side.
 
+See [victoria-metrics-distributed chart](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-distributed) for an example.
```

Comment:
It is better to use link to our docs, but I don't see distributed chart there for single https://docs.victoriametrics.com/helm